# HG changeset patch
# User Rik <rik@octave.org>
# Date 1357328468 28800
#      Fri Jan 04 11:41:08 2013 -0800
# Node ID a7d89366d7edf03cfe602c1201c18b38a4fca032
# Parent  9cd14e53e906860fff663edaec7d922f553f6f02
Document that break and continue statements work for do-until loops.

* doc/interpreter/stmt.txi: Document that break and continue statements work
for do-until loops.

diff --git a/doc/interpreter/stmt.txi b/doc/interpreter/stmt.txi
--- a/doc/interpreter/stmt.txi
+++ b/doc/interpreter/stmt.txi
@@ -657,20 +657,21 @@ The @var{key} variable may also be omitt
 also optional.  This is useful for cycling through the values of all the
 structure elements when the names of the elements do not need to be
 known.
 
 @node The break Statement
 @section The break Statement
 @cindex @code{break} statement
 
-The @code{break} statement jumps out of the innermost @code{for} or
-@code{while} loop that encloses it.  The @code{break} statement may only
-be used within the body of a loop.  The following example finds the
-smallest divisor of a given integer, and also identifies prime numbers:
+The @code{break} statement jumps out of the innermost @code{while},
+@code{do-until}, or @code{for} loop that encloses it.  The @code{break}
+statement may only be used within the body of a loop.  The following
+example finds the smallest divisor of a given integer, and also
+identifies prime numbers:
 
 @example
 @group
 num = 103;
 div = 2;
 while (div*div <= num)
   if (rem (num, div) == 0)
     break;
@@ -713,19 +714,20 @@ endwhile
 @end group
 @end example
 
 @node The continue Statement
 @section The continue Statement
 @cindex @code{continue} statement
 
 The @code{continue} statement, like @code{break}, is used only inside
-@code{for} or @code{while} loops.  It skips over the rest of the loop
-body, causing the next cycle around the loop to begin immediately.
-Contrast this with @code{break}, which jumps out of the loop altogether.
+@code{while}, @code{do-until}, or @code{for} loops.  It skips over the
+rest of the loop body, causing the next cycle around the loop to begin
+immediately.  Contrast this with @code{break}, which jumps out of the
+loop altogether.
 Here is an example:
 
 @example
 @group
 # print elements of a vector of random
 # integers that are even.
 
 # first, create a row vector of 10 random
